scripts/perftune.py

```diff
@@ -606,6 +608,18 @@ def tune(self):
         # did not receive an acknowledgment from connecting client.
         fwriteln_and_log('/proc/sys/net/ipv4/tcp_max_syn_backlog', '4096')
 
+        self.tune_tcp_mem()
+
+    def tune_tcp_mem(self):
+        page_size = mmap.PAGESIZE
+        total_mem = psutil.virtual_memory().total
+        # We only tune for physical memory since tcp_mem is virtualized
+        def to_pages(bytes):
+            return math.ceil(bytes / page_size)
+        max = total_mem * self.args.tcp_mem_fraction
+        with open('/proc/sys/net/ipv4/tcp_mem', 'w') as f:
```

Comment:
Other sysctls are tuned using `fwriteln_and_log()` helper, it's worth using one here as well

Reply:
Yes. It's also a way to prepare an "undo file" which can be applied on service shutdown. @vladzcloudius

Reply:
That was a good point, @xemul. I forgot about that wrapper.
It's used by `--dry-run`.